Resources/config/form-type.xml

```diff
@@ -9,5 +9,13 @@
             <tag name="form.type" alias="cmf_core_checkbox_url_label" />
             <argument type="service" id="router" />
         </service>
+
+        <service id="cmf_core.form.extension.publishable" abstract="true" class="Symfony\Cmf\Bundle\CoreBundle\Form\Extension\PublishableExtension">
+            <argument></argument><!-- Must be overrided when using this service as parent -->
```

Comment:
Contributing to Symfony, will imply to not release cmf 2.0 before april 2017. I don't know how much work is required to get cmf 2.0 out, but we should take it in consideration!